sdk/js/rollup.config.mjs

```diff
@@ -0,0 +1,28 @@
+import alias from '@rollup/plugin-alias'
+import resolve from '@rollup/plugin-node-resolve'
+// Require other plugins you might be using
+
+module.exports = {
+    output: [
+        {
+            file: 'dist/sdk/js/index.js',
+            format: 'cjs',
+        },
+        {
+            file: 'dist/sdk/js/index.esm.js',
+            format: 'esm',
+        }
+    ],
+    plugins: [
+        // Other plugins...
+        resolve(), // Make sure resolve plugin is used
+        alias({
+            entries: [
+                { find: '@devcycle/server-request', replacement: 'lib/shared/server-request/src/index.ts' },
```

Comment:
this is really sketchy...and seems like it would definitely cause issues with Nx.

Reply:
yeah, this isnt supposed to be here, didnt mean to push this config up. Apparently overriding the rollup config is the only way to fix the issue im facing tho so Im not sure what to do